edgelet/build/linux/docker/cross-compile/Dockerfile.ubuntu18.04.aarch64-unknown-linux-gnu

```diff
@@ -0,0 +1,49 @@
+FROM ubuntu:18.04
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends \
+    git \
+    software-properties-common \
+    build-essential \
+    ca-certificates \
+    uuid-dev \
+    curl \
```

Comment:
I remember curl is available in ubuntu:18.04 image.  is it a common practice to put all required software?

Reply:
Asking for a dependency it already has isn't going to impact anything - I think it's probably better to spell it out.